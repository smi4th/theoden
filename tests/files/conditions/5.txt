a=2;
if (a == 1) {
    a=2;
} else { if (a==2) {
    a=3;
} else {
    a=4;
} }