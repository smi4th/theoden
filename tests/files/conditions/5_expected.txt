---TEST OUTPUT---
---MEMORY---
a = 3
---FUNCTIONS---
